src/client/Microsoft.Identity.Client/AppConfig/ConfidentialClientApplicationBuilder.cs

```diff
@@ -1,340 +1,341 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using System;
```

Comment:
What are your spacing settings in Visual Studio? You should be using the defaults - 4 spaces. 

I think when we make changes via GitHub (via those suggestions), it messes up spacing and makes this impossible to review.

Reply:
Anyway, if you see this happening, I think the best is to create a commit that changes the spacing and then another commit with actual changes, to help reviewers.

Reply:
Both files seem to have 4-space tabs. This might be the Mac/Linux vs Windows new line encodings?